import-scripts/anonymize_age_at_seq_with_cap_py3.py

```diff
@@ -0,0 +1,421 @@
+        with open(self.input_file_path, 'r') as input_file_handle:
+            with open(self.output_file_path, 'w') as output_file_handle:
+                line_processor = SampleLineProcessor(
+                    self.patient_os_years_map,
+                    self.col_indices,
+                    output_file_handle,
+                    self.upper_age_limit,
+                    self.lower_age_limit,
+                )
+                for line in input_file_handle:
+                    line_processor.process(line)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='transform a clinical sample file so that specified age limits are applied to age_at_seq_reported'
+    )
+    parser.add_argument('input_patient_file_path', help='Path to the input clinical patient file')
+    parser.add_argument('output_patient_file_path', help='Path to which the output patient file is written')
+    parser.add_argument('input_sample_file_path', help='Path to the input clinical sample file')
+    parser.add_argument('output_sample_file_path', help='Path to which the output sample file is written')
+    parser.add_argument(
+        '--upper-age-limit',
+        '-u',
+        dest='upper_age_limit',
+        help='ages above this (integer) limit will be replaced with ">{upper_age_limit}"',
+        required=False,
+        default=89,
+        type=int,
+    )
+    parser.add_argument(
+        '--lower-age-limit',
+        '-l',
+        dest='lower_age_limit',
+        help='ages below this (integer) limit are not expected and will cause a thrown exception',
+        required=False,
+        default=18,
+        type=int,
+    )
+    parser.add_argument(
+        '--os-months-precision',
+        '-o',
+        dest='os_months_precision',
+        help='Digits of precision for which to specify OS_MONTHS attribute',
+        required=False,
+        default=2,
+        type=int,
+    )
+
+    args = parser.parse_args()
+
+    input_patient_file_path = args.input_patient_file_path
+    output_patient_file_path = args.output_patient_file_path
+    input_sample_file_path = args.input_sample_file_path
+    output_sample_file_path = args.output_sample_file_path
+    upper_age_limit = args.upper_age_limit
+    lower_age_limit = args.lower_age_limit
+    os_months_precision = args.os_months_precision
+
+    if upper_age_limit < 0:
+        raise ValueError('upper_age_limit is negative')
+    if lower_age_limit < 0:
+        raise ValueError('lower_age_limit is negative')
+    if os_months_precision < 0:
+        raise ValueError('os_months_precision is negative')
+
+    # Transform the patient file
+    patient_file_reader = PatientFileWriter(
+        input_patient_file_path, output_patient_file_path, upper_age_limit, lower_age_limit, os_months_precision
+    )
+    patient_file_reader.write()
+
+    # Transform the clinical sample file
+    sample_file_writer = SampleFileWriter(
+        input_sample_file_path,
```

Comment:
The style of argument passing should match between here and the patient file reader above. If this is preferred then make the declaration of patient_file_reader match.